File: app/src/main/java/com/prox/voicechanger/utils/ColorUtils.java
package com.prox.voicechanger.utils;

import android.graphics.LinearGradient;
import android.graphics.Shader;

public class ColorUtils {
    public static Shader textShader(int startColor, int endColor, float height){
        return new LinearGradient(
                0,
                0,
                0,
                (float) (height*1.3),
                startColor,
                endColor,
                Shader.TileMode.CLAMP);
    }
}

File: app/src/main/java/com/prox/voicechanger/utils/ColorUtils.kt
package com.prox.voicechanger.utils

import android.graphics.LinearGradient
import android.graphics.Shader

object ColorUtils {
    fun textShader(startColor: Int, endColor: Int, height: Float): Shader {
        return LinearGradient(
            0f,
            0f,
            0f, (height * 1.3).toFloat(),
            startColor,
            endColor,
            Shader.TileMode.CLAMP
        )
    }
}